frontend/lib/utils/credential-types.ts

```
@@ -0,0 +1,180 @@
+		case 'password':
+			return { username: credential.username, password: credential.password }
+		case 'ssl_cert':
+			return { 
+				certificateFile: credential.certificateFile, 
+				certificateFileName: credential.certificateFileName 
+			}
+		case 'api_key':
+			return { apiKey: credential.apiKey }
+		default:
+			// This should never happen with the discriminated union
+			const _exhaustive: never = credential
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)